docs/installation.rst

```diff
@@ -1,51 +1,35 @@
-.. highlight:: shell
-
 ============
 Installation
 ============
 
-
-Stable release
---------------
-
-To install qmxgraph, run this command in your terminal:
-
-.. code-block:: console
-
-    $ pip install qmxgraph
-
-This is the preferred method to install qmxgraph, as it will always install the most recent stable release. 
-
-If you don't have `pip`_ installed, this `Python installation guide`_ can guide
-you through the process.
-
-.. _pip: https://pip.pypa.io
-.. _Python installation guide: http://docs.python-guide.org/en/latest/starting/installation/
-
-
-From sources
-------------
-
-The sources for qmxgraph can be downloaded from the `Github repo`_.
-
-You can either clone the public repository:
-
-.. code-block:: console
-
-    $ git clone git://github.com/ESSS/qmxgraph
-
-Or download the `tarball`_:
-
-.. code-block:: console
-
-    $ curl  -OL https://github.com/ESSS/qmxgraph/tarball/master
-
-Once you have a copy of the source, you can install it with:
-
-.. code-block:: console
-
-    $ python setup.py install
-
-
-.. _Github repo: https://github.com/ESSS/qmxgraph
-.. _tarball: https://github.com/ESSS/qmxgraph/tarball/master
+Since pip no longer offer the pyqt version required by qmxgraph you will need `Conda`_ and `conda-devenv`_ to setup the environment.
```

Comment:
```suggestion
Since pip no longer offers the PyQt version required by qmxgraph you will need `Conda`_ and `conda-devenv`_ to setup the environment.
```